CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+Here is some information that's good to know when contributing to JavaParser.
+
+- if you're new and like to contribute something, check out the [easy issues](https://github.com/javaparser/javaparser/labels/Easy)
+- we work on JavaParser because we like to, not because it earns us money. Please remember that we try to run a professional project in our spare time, on a budget of zero.
+- if you know how to fix a problem, please fix it and open a pull request instead of opening an issue.
+- a pull request should include tests. You can either use BDD (JBehave) or JUnit.
+- there is some interesting information on the [wiki](https://github.com/javaparser/javaparser/wiki).
+- Be sure to check [the coding guidelines](https://github.com/javaparser/javaparser/wiki/Coding-Guidelines) which are easily used by installing the formatting rules as described there.
+- pull requests often stay open for at least a few days to give people a chance to review it.
+- a pull request is merged when all comments on it have been resolved. 
```

Comment:
Should we specify that Travis and AppVeyor must be happy? It should be obvious but...